src/mlpack/methods/ann/layer/adaptive_mean_pooling.hpp

```diff
@@ -46,13 +48,21 @@ class AdaptiveMeanPoolingType : public Layer<InputType, OutputType>
   AdaptiveMeanPoolingType(const size_t outputWidth,
                           const size_t outputHeight);
 
-  /**
-   * Create the AdaptiveMeanPooling object.
-   *
-   * @param outputShape A two-value tuple indicating width and height of the
-   *      output.
-   */
-  AdaptiveMeanPoolingType(const std::tuple<size_t, size_t>& outputShape);
+  // Virtual destructor.
+  virtual ~AdaptiveMeanPoolingType() { 
```

Comment:
```suggestion
  virtual ~AdaptiveMeanPoolingType()
  { 
```

To match with the style guidelines.

Reply:
Done.